### ISSUE TITLE
Example of creating an entry with a field that links to multiple entries?

### ISSUE THREAD

**Issue by jonsie** [contentful/contentful-management.java#27]
Hello,

I can't seem to figure out how to create an entry with a field that links to multiple entries.  I checked out the tests but there isn't an example of how this might work.  Any hints?  :)

**Comment by tomxor** [comment]
Currently the CMA SDK has no support for link resolution, which is probably why you can't seem to get it to work :) when setting a field on an entry it is simply converted to JSON before being posted to the server, so right now the entire entry is being posted to the API. Since the API expects a specific format for links this currently fails.

We are planning another major release for both the CDA and CMA SDKs pretty soon which should have this issue covered also, for the meantime consider the following workaround:

```java
CMAClient client = new CMAClient.Builder()
    .setAccessToken("token")
    .build();

final String CHILD_CONTENT_TYPE = "...";
final String PARENT_CONTENT_TYPE = "...";
final String SPACE_ID = "...";
final String LOCALE = "...";

CMAEntry child1 = client.entries().create(SPACE_ID, CHILD_CONTENT_TYPE,
    new CMAEntry().setField("name", "child-1", LOCALE));

CMAEntry child2 = client.entries().create(SPACE_ID, CHILD_CONTENT_TYPE,
    new CMAEntry().setField("name", "child-2", LOCALE));

List children = Arrays.asList(resourceToLink(child1), resourceToLink(child2));

CMAEntry parent = client.entries().create(SPACE_ID, PARENT_CONTENT_TYPE,
    new CMAEntry().setField("children", children, LOCALE));

static Map<String, ?> resourceToLink(CMAResource resource) {
  final String resourceId = resource.getResourceId();
  if (resourceId == null) {
    throw new IllegalArgumentException("Called for a resource that has no ID!");
  }

  Map<String, String> sys = new HashMap<String, String>();
  sys.put("type", "Link");
  sys.put("linkType", (String) resource.getSys().get("type"));
  sys.put("id", resourceId);

  Map<String, Object> result = new HashMap<String, Object>();
  result.put("sys", sys);
  return result;
}
```

`resourceToLink()` simply converts any `CMAResource` to the desired format for links.
Please let me know if this solution is working for you.

**Comment by jonsie** [comment]
Oh great Tom, you rule.  This workaround will totally be fine for now.  Thank you!

**Comment by tomxor** [comment]
No worries, glad I could help!

**Comment by tomxor** [comment]
@jonsie this is now available in v0.9.9, here's an [example test case](https://github.com/contentful/contentful-management.java/blob/master/src/test/kotlin/com/contentful/java/cma/EntryTests.kt#L97-L116).

**Comment by jonsie** [comment]
@tomxor awesome - nice work!
